src/main/webapp/WEB-INF/jsp/oneclick_mall_deferred/aborted.jsp

```diff
@@ -0,0 +1,25 @@
+<head>
+    <meta charset="UTF-8">
+
+</head>
+<jsp:include page="../template/tpl-header.jsp" />
+
+<body class="container">
+    <jsp:include page="../template/navbar-oneclick.jsp" />
+
+<style>
+	.row {
+		padding-bottom: 20px;
+	}
+</style>
+
+<div class="row">
+	<div class="col">
+		<h2>Inscripci�n cancelada</h2>
```

Comment:
This will appear as `ó` when the page is rendered? asking because I see a � symbol